docs/howtos/docker.md

```diff
@@ -4,7 +4,9 @@
 
 ## Build
 
-Builds are available in the [GitHub Container Registry](https://github.com/orgs/napari/packages).
+Builds are available in the GitHub Container Registry:
+- [napari](https://github.com/napari/napari/pkgs/container/napari)
+- [napari-xpra](https://github.com/napari/napari/pkgs/container/napari-xpra)
 
 A dockerfile is added to napari root to allow build of a docker image using official napari release.
 It contains two targets built on top of Ubuntu 20.04:
```

Comment:
```suggestion
It contains two targets built on top of the official Python 3.11 slim-bookworm image:
```
I'm not sure this is important. Maybe we just delete it so it's not something that needs to be maintained?

Similarly, below it should say:
* `napari`: The result of `python -m pip install pip install "napari[all]", including all the ...

Reply:
Updating this is on my todo list 👀 (along with updating the xpra container), so I'd not worry about the whole document. But I just noticed those two lines as being out-dated. We could just remove them.